lib/internal/errors.js

```diff
@@ -737,6 +740,8 @@ E('ERR_NO_ICU', '%s is not supported on Node.js compiled without ICU');
 E('ERR_NO_LONGER_SUPPORTED', '%s is no longer supported');
 E('ERR_OUT_OF_RANGE', outOfRange);
 E('ERR_PARSE_HISTORY_DATA', 'Could not parse history data in %s');
+E('ERR_PATH_IS_DIRECTORY',
+  'Path %s cannot end with / in operations invalid on directories');
```

Comment:
Can you use a custom function here for not displaying Uint8Arrays?

Reply:
> Does the test pass locally?

Yes. it does.